js/stdlib/ts/cfxers.ts

```diff
@@ -282,3 +274,93 @@ export class Wallet {
     throw Error(`Account 'from mnemonic' not supported on Conflux, please use secret key`);
   }
 }
+
+// XXX This is nutty
+// Remember the last epoch that a given sender has sent
+// and don't try to send again until it is later than that epoch.
+// Note: requires addrs to be canonicalized first.
+const lastEpochSent: Record<string, number> = {};
+const epochWaitLock: Record<string, boolean> = {};
+
+// XXX implement a queue, maybe?
```

Comment:
I think we can and should be more willing to require other npm packages. There must be a good thread-safe JS queue